src/oogeso/utils/util.py

```diff
@@ -1,19 +1,49 @@
-from oogeso.dto.oogeso_input_data_objects import TimeSeriesData
-from typing import List
 import logging
+from typing import Callable, List
+
 import pandas as pd
 
+from oogeso import dto
+from oogeso.core import devices
+
 logger = logging.getLogger(__name__)
 
 
-def create_timeseriesdata(
+def get_device_from_model_name(model_name: str) -> Callable:
```

Comment:
Would it be better/better to have an explicit map (dictionary)
device_name_to_class_name_map = {
"powersink" : devices.Powersink,
"source_el" : devices.SourceEl,
...
}

It's simpler to understand. But the good thing with the present code is that if we add a device type we don't have to change this code.

Reply:
Yes, much better :) 

That is why I added this: https://github.com/oogeso/oogeso/issues/54